testsuite/testsuite_impl/my_test/my_test_.go

```diff
@@ -44,169 +20,23 @@ func (test MyTest) Configure(builder *testsuite.TestConfigurationBuilder) {
 }
 
 func (test MyTest) Setup(networkCtx *networks.NetworkContext) (networks.Network, error) {
-	containerCreationConfig, runConfigFunc := getEthereumServiceConfigurations()
 
-	serviceContext, hostPortBindings, err := networkCtx.AddService(serviceID, containerCreationConfig, runConfigFunc)
-	if err != nil {
-		return nil, stacktrace.Propagate(err, "An error occurred adding the Ethereum Go Client service")
-	}
+	//TODO Replace with the first part of the implementation
```

Comment:
Might be worth making these more specific - e.g. "Replace with instructions for starting the ETH network" (or something - whatever the correct part to slot in is)